common/src/main/java/org/apache/hertzbeat/common/entity/message/CollectRep.java

```diff
@@ -449,7 +449,7 @@ public static final class MetricsData extends
         private MetricsData(com.google.protobuf.GeneratedMessageV3.Builder<?> builder) {
             super(builder);
         }
-        private MetricsData() {
+        public MetricsData() {
```

Comment:
hi, the `CollectRep` is auto geneate by protocol buffer, can not be edit.

Reply:
sorry, my bad. I will fix it.